Swap tf to match rotation axis of scanner (#57)
PilzDE/psen_scan_v2: README.md

_sensor_ip_ (_string_, default: "192.168.0.10")<br/>
IP-Address of safety laser scanner.

> _prefix_ (_string_, default: "laser_1")<br/>

Reviewer: is this a prefix also used by other vendors or could we name it like psenscan_1 to have it related to the pilz product?

Author: other vendors use a similar method. I think we have the psenscan branding already in many places.

```
This example configures the safety laser scanner at 192.168.0.10 to send it´s frames to 192.168.0.20:3050.

## TF Frames
The location of the TF frames is shown in the image below.

> Note that these names defined by the `prefix` parameter.
> This can be changed with aforementioned launchfile parameters.
> For example when you are running multiple scanners.

Reviewer: ```suggestion
> These names are defined by the aforementioned launchfile parameter: `prefix` \
> Changing them is necessary for instance when running multiple scanners.
```

<p align="center">
<img src="img/frames.png" width="800px" alt="PILZ safety laser scanner frames" title="frames">

> </p>

Reviewer: I think side and rear view already give all necessary information for this topic. Maybe omit the other two at this point?

Author: I like them for clarity
